nbs/sidebar.yml

```diff
@@ -17,6 +17,7 @@ website:
           - docs/prediction_intervals.ipynb
           - docs/transfer_learning.ipynb
           - docs/target_transforms_guide.ipynb
+          - docs/Prediction_intervals_in_forecasting_models.ipynb
```

Comment:
Please rename this notebook to have all lowercase letters (prediction_intervals_in_forecasting_models)